internal/hns/endpointstate_string_windows.go

```diff
@@ -0,0 +1,29 @@
+// Code generated by "stringer -type=EndpointState -output=endpointstate_string_windows.go -trimprefix=EndpointState -tags=windows"; DO NOT EDIT.
+
```

Comment:
I am not sure we need to use stringer here - we only want to maintain a state field which have 7 possible states. We can define them in hcsendpoint.go and also add the field in the struct as needed.

Reply:
That is a sensible point, updated the code to have its own string function which can be updated if fields change to avoid bloat on hcsshim.